source/main/gameplay/BeamEngine.cpp

```diff
@@ -248,7 +291,7 @@ void BeamEngine::update(float dt, int doUpdate)
 	if (automode < MANUAL)
 	{
 		// auto-shift
-		if (shifting)
+		if (shifting && !is_Electric) //No shifting in electric cars

```

Comment:
This is wrong. I'd say direct wheel drive vehicles (which means they don't have a gearbox) are in the minority.